player/javascript.c

```diff
@@ -1256,7 +1256,7 @@ static void add_functions(js_State *J, struct script_ctx *ctx)
 
 // main export of this file, used by cplayer to load js scripts
 const struct mp_scripting mp_scripting_js = {
-    .name = "javascript",
+    .name = "js",
```

Comment:
I think keeping this as `javascript` is better since it's not actually related to the thread names.

Reply:
It actually is the one place that it is used as a backend name in thread names.

Reply:
Oh sorry, I somehow missed that.